conda_build/jinja_context.py

```diff
@@ -167,6 +167,43 @@ def load_npm():
         return json.load(pkg)
 
 
+def load_file_regex(config, load_file=None, regex_pattern=None, from_recipe_dir=False,
+                    recipe_dir=None, permit_undefined_jinja=True):
+    import re
+    match = False
+
+    cd_to_work = False
+
+    if load_file is None:
+        raise RuntimeError('File to be searched is not specified.')
+
+    if from_recipe_dir and recipe_dir:
+        load_file = os.path.abspath(os.path.join(recipe_dir, load_file))
+    elif os.path.exists(config.work_dir):
```

Comment:
This error was because the rendering happens in multiple passes.  In the first pass, the source has not been downloaded, so your exception gets raised.  It's import to not raise unless undefined jinja is not allowed.

Fixing this in incoming PR.